common/core/desktop/tests/unit/kmx/041 - long context and deadkeys.kmn

```diff
@@ -22,4 +25,7 @@ c Test 3: Test that we haven't split a deadkey code: UC_SENTINEL CODE_DEADKEY <d
 c         because: CODE_DEADKEY = U+0008; deadkey_number = U+0001
 U+0008 U+0001 'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh12' + '3' > 'FAIL TEST 3'
 U+0001 'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh12' + '3' > 'FAIL TEST 3'
-'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh12' + '3' > context '3'
+'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh12' + '3' > 'abcdefghijklmnopqrstuvwxyz123'
+
+c Convert to a visible result:
+dk(1) 'abcdefghijklmnopqrstuvwxyz123' + '4' > 'PASS'
```

Comment:
This changes the meaning of this test. If we get to MAXCONTEXT, that means that additional content at the start of the cached context should be chopped off. This means that the `dk(1)` should be permanently lost, so if it isn't it probably means that we are not testing the context boundary as we think it should be.

In fact, this final test of `[K_4]` should be inverted: if `dk(1)` is still there, then we haven't tested what we want so it should fail... 😆:

```
c deadkey should have been trimmed from cached context;
c if it is not then we have not reached the cached context
c boundary!
dk(1) 'abcdefghijklmnopqrstuvwxyz123' + '4' > 'FAIL'
c This should match if the above rule doesn't.
'abcdefghijklmnopqrstuvwxyz123' + '4' > 'PASS'
```

Reply:
And we do fail...